tox.ini

```diff
@@ -32,3 +32,6 @@ commands = bashate --verbose {toxinidir}/build.sh {toxinidir}/tools/test/functes
 
 [testenv:functional]
 commands = {toxinidir}/tools/test/functest.sh
+
+[testenv:mypy]
+commands = -mypy hotsos
```

Comment:
looking at the ci job output, mypy is complaining about `error: Library stubs not installed for "yaml"  [import]`. The official [documentation](https://mypy.readthedocs.io/en/stable/running_mypy.html#library-stubs-not-installed) is suggesting to run mypy with `--install-types --non-interactive` flags to automatically install the stubs as needed.

Reply:
Thanks for the hint @mustafakemalgilor ! Fixed.